### ISSUE TITLE
ERROR like [rtmp @ 0x7f1b147cbb40] Too short chunk size change packet (0)

### ISSUE THREAD

**Issue by NacyLiu** [bytedeco/javacv#469]
@saudet  I have used FFmpegFrameGrabber to read video from a rtmp stream, and use grabImage() to grab frame, but it raised [rtmp @ 0x7f1b147cbb40] Too short chunk size change packet (0), how can i catch it ?

**Comment by saudet** [comment]
I'm sorry I don't understand your explanations. Please explain more clearly what the issue is and what you are trying to accomplish.

**Comment by NacyLiu** [comment]
i want to grab images from a rtmp stream, here is my code...

FFmpegFrameGrabber grabber = null;
 try {
    grabber = FFmpegFrameGrabber.createDefault(url);
} catch (Exception e) {
    logger.error(e);
}
....
try {
            videoFrame = grabber.grabImage();
} catch (Exception e) {
            logger.error(e);
}
.....

and the error raised about 10 hours later, dose it cause by network error ? or others ... i am not sure, but i want to find it and fix it ....

**Comment by saudet** [comment]
Try to use ffmpeg or ffplay on the command line. If you get the same error, then it's probably an issue with the network, or a bug in FFmpeg. If it's a bug in FFmpeg, please post a bug report upstream, and please copy the URL in this issue for reference. Thank you

**Comment by NacyLiu** [comment]
ok, i will try, thank you

**Comment by NacyLiu** [comment]
i was told that the chunk_size can be changed in rtmplib(C/C++), but i cannot find it in FFmpegFrameGrabber, is the place i looked for wrong ?

**Comment by saudet** [comment]
maybe `setOption("chunk_size", "XYZ")`?
